examples/nlp/intent_detection_slot_tagging/joint_intent_slot_infer.py

```diff
@@ -18,14 +18,15 @@
 import os
 
 import numpy as np
-from sklearn.metrics import classification_report
+from sklearn.metrics import confusion_matrix
 
 import nemo
 import nemo.collections.nlp as nemo_nlp
 from nemo import logging
 from nemo.collections.nlp.data.datasets.joint_intent_slot_dataset import JointIntentSlotDataDesc
 from nemo.collections.nlp.nm.data_layers import BertJointIntentSlotDataLayer
 from nemo.collections.nlp.nm.trainables.joint_intent_slot import JointIntentSlotClassifier
+from nemo.collections.nlp.utils.callback_utils import get_classification_report, get_f1_scores
 
 # Parsing arguments
 parser = argparse.ArgumentParser(description='Batch inference for intent detection/slot tagging with BERT')
```

Comment:
parser should go to main()

Reply:
All our code in this package is not written in this way. I do not want to make too big refactoring at this stage.